fields/types/password/PasswordColumn.js

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+
+var PasswordColumn = React.createClass({
+
+	render: function() {
+		var value = this.props.data.fields[this.props.col.path];
+		if (value) {
+			return (
+				<td>
+					<p>******</p>
```

Comment:
For most columns, you want this inside the `<td>`:

```
<div className="col-value">{value}</div>
```

the `col-value` class is important for styling.

Checkboxes were an exception